### ISSUE TITLE
Wrong line-height in Day and Week view

### ISSUE THREAD

**Issue by DanielvV** [owncloud/calendar#616]
When you click on a late hour in the *day* or *week* view the event is created with a half an hour time shift.

### Steps to reproduce
1. Open Firefox
2. [Go to the ownCloud calendar demo] (http://demo.owncloud.org/index.php/apps/calendar)
3. Click on the *Day* or *Week* button.
4. Click or drag on a late hour
5. *The event is being created with a half an hour time shift*

Applies to:
Calender, Day and Week view
Firefox (also IE8)
Doesn't apply to:
Chromium

My suggestion:
In the file *owncloud/apps/calendar/css/style.css*:
```css
.fc-agenda .fc-agenda-axis{
        padding:0;
        padding-right:4px;
        line-height:initial; /* my suggestion for a fix for Firefox etc */
}
```
### Client configuration
**Browser:** Firefox

**Comment by MorrisJobke** [comment]
Fixed with #617

**Comment by DanielvV** [comment]
Thanks!
This was my first pull request ever after my first comment ever on GitHub.
